tests/smoke/recommenders/recommender/test_newsrec_model.py

```diff
@@ -115,7 +115,7 @@ def test_model_naml(mind_resource_path):
     assert model.run_eval(valid_news_file, valid_behaviors_file) is not None
     assert isinstance(
         model.fit(
-            train_news_file, train_behaviors_file, valid_news_file, valid_behaviors_file
+            train_news_file, train_behaviors_file, valid_news_file, valid_behaviors_file,steps=1
```

Comment:
can you check how long it takes with step limit of 1 versus 5 or other? it would be good if you share the details

Reply:
I have checked time taken for different step_limit, here are the outputs

for step_limit = 1

>> Output:
[Done] exited with code=0 in 0.614 seconds

for step_limit = 3
>> Output:
[Done] exited with code=0 in 0.677 seconds 

for step_limit = 5
>> Output:
[Done] exited with code=0 in 0.984 seconds

for step_limit = 7
>> Output:
[Done] exited with code=0 in 0.488 seconds

for step_limit = 10
>> Output:
[Done] exited with code=0 in 0.846 seconds

- step_limit=7 has taken lesser time than 1